packages/terra-cell-grid/src/CellGrid.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './CellGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Indicates the desired divider styles to apply to a cell grid and its children.
+   * One of `'none'`, `'vertical'`, `'horizontal'`, `'both'`.
```

Comment:
props table will handle the 'one of' stuff now, you don't have to include it in the description anymore if you don't want.

Reply:
Updated.